cnd/Cargo.toml

```diff
@@ -63,6 +63,7 @@ memsocket = "0.1"
 pretty_env_logger = "0.3"
 quickcheck = "0.8.5"
 quickcheck_macros = "0.8.0"
+regex = "1.2.0"
```

Comment:
Didn't we say we prefer no to specify the patch version?

Reply:
Hmm, that was pre-dependabot though. Does it still matter now?

Reply:
I don't know, you're the one who came up with the previous rule...